youtube_dl/extractor/cbsinteractive.py

```diff
@@ -89,6 +98,23 @@ def _real_extract(self, url):
             tp_formats, tp_subtitles = self._extract_theplatform_smil(release_url, video_id, 'Downloading %s SMIL data' % fkey)
             formats.extend(tp_formats)
             subtitles = self._merge_subtitles(subtitles, tp_subtitles)
+
+        if 'm3u8' in vdata:
+            parsed_url = compat_urllib_parse.urlparse(url)
+            m3u8_url = ('%s://%s%s'
+                % (parsed_url.scheme, parsed_url.netloc, vdata['m3u8']))
+            m3u8_formats = self._extract_m3u8_formats(m3u8_url, video_id)
+            for format in m3u8_formats:
+                format['url'] = format['url'].replace('https://', 'http://')
```

Comment:
Extracted `m3u8` formats have URL starting with `https://ipad-streaming.cnet.com/` and their certificate doesn't apply to that host. This can be fixed by adding `?change-to-host=cnetmedia-a.akamaihd.net` to master `m3u8` URL but I don't really know how to extract proper host. And it can't be hardcoded because `CBSInteractiveIE` extracts videos from `cnet.com` and `zdnet.com`, which most probably has different host. Dropping `https` works as dirty workaround.